src/framework/parsers/glb-parser.js

```diff
@@ -1804,17 +1804,17 @@ const createScenes = function (gltf, nodes) {
     return scenes;
 };
 
-const createCameras = function (gltf, nodes, options) {
+const createCameras = (gltf, nodes, options) => {
 
     let cameras = null;
 
     if (gltf.hasOwnProperty('nodes') && gltf.hasOwnProperty('cameras') && gltf.cameras.length > 0) {
 
-        const preprocess = options && options.camera && options.camera.preprocess;
-        const process = options && options.camera && options.camera.process || createCamera;
-        const postprocess = options && options.camera && options.camera.postprocess;
+        const preprocess = options?.camera?.preprocess;
```

Comment:
Wondering if we should prescribe a set coding style for options objects. In my code, I've been using a default parameter (empty object) for options objects. But, of course, that will allocate on every call potentially. Whereas using `?.` everywhere is a bit more verbose and might transpile to more code via Babel?

Reply:
Not sure how that would work here. Could you give a code snippet?

Reply:
Well, you'd still need the second `?.` regardless. TBH, it's not a big deal.